src/examples/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ set(BASIC_EXAMPLES
     variorum-set-gpu-power-ratio-example
     variorum-set-node-power-limit-example
     variorum-set-socket-power-limits-example
+    variorum-json-get-node-power-example
```

Comment:
Can we move `-json-` to `-json-example`? I think it'll help with the sorting of the examples.